### DISCUSSION TITLE
Can't see full action name without mouseover

### DISCUSSION THREAD

**Discussion by ibrunotome** [laravel/nova-issues#4252] [Ideas & Feature Requests]
- Nova Version: 4.5.2
- Operating System and Version: macOS 12
- Browser type and version: Tested with Safari 15.4 and Chrome 101.0.4951.64

### Description:
I have some descriptive names for the actions in the details screen, they were displayed in full size in Nova v3, but in v4 we need to mouseover to see the full action name:

<img width="253" alt="Screen Shot 2022-05-13 at 08 24 59" src="https://user-images.githubusercontent.com/4256471/168274495-911f581a-7fb2-4377-bb28-7f20991cdc2c.png">

**Comment by ibrunotome** [comment]
@davidhemphill @crynobone can't this be customizable? I'm with dozens of actions and can't read full description without mouseover and scroll:

<img width="353" alt="Screen Shot 2022-05-26 at 22 20 15" src="https://user-images.githubusercontent.com/4256471/170609488-4c769adf-5dde-476e-a932-4a3de03cd51b.png">

**Comment by lucacri** [comment]
I have the same problem. It's almost necessary to have the option to customize the size, or at least allow a "don't truncate" flag

**Comment by davidhemphill** [comment]
At what point should we truncate this? This seems kinda silly:

![image](https://user-images.githubusercontent.com/58970/174132591-eca20915-f769-42e2-a6f8-fd2f2c50b426.png)

**Reply by ibrunotome** [reply]
IMHO the container should be: `max-w-full lg:max-w-lg`

**Reply by davidhemphill** [reply]
We've added a change that makes the action dropdown expand to accommodate longer action names. Will be in the next release!

**Reply by ibrunotome** [reply]
Installed the v4.10.0, published the assets, but still like before, do I have anything more to do? (didn't see mentions in the doc)

<img width="274" alt="Screen Shot 2022-06-21 at 18 15 35" src="https://user-images.githubusercontent.com/4256471/174898683-6e1f2c7f-7852-4703-8972-8a852a381221.png">

**Reply by ibrunotome** [reply]
Inspecting the html, there are inline styles with fixed width:

<img width="1020" alt="Screen Shot 2022-06-21 at 18 27 16" src="https://user-images.githubusercontent.com/4256471/174900199-a6d634da-c7df-4e19-b270-cd0846420eb1.png">

**Reply by ibrunotome** [reply]
@lucacri @juse-less did it worked for you guys?

**Reply by juse-less** [reply]
~~For me it still truncates on the same places as before. Both locally and on our different server environments.
I updated all our npm- and Composer dependencies (including Nova v4.10), and deployed earlier today. Not sure if I've done anything wrong, though.
But I have run the regular publishing that Laravel also comes with by default nowadays (for the post-update-cmd script).~~

~~FWIW: I quickly tried installing Novas npm dependencies and built and bundled them, as well as re-publishing afterwards locally, to no avail.~~

Edit: Ignore what I wrote. I'm a moron. 🤦‍♀️ 
My problem isn't actions (never been). My problem is with lenses, which seems to stem from the class values in components/Dropdowns, that it uses, unlike the action component.

**Reply by davidhemphill** [reply]
Just pushed an update for the detail view dropdown too. Will probably release a similar update for the lens dropdown on Tuesday.

**Reply by ibrunotome** [reply]
Tested v4.10.1

Safari:
![telegram-cloud-photo-size-1-5019282942169557654-x](https://user-images.githubusercontent.com/4256471/175343313-7301bd6d-b2cb-43dc-b2b7-15560ad1387d.jpg)

Chrome:
<img width="457" alt="Screen Shot 2022-06-23 at 13 07 31" src="https://user-images.githubusercontent.com/4256471/175344877-660760db-6029-46a8-b38a-16c6d20b160e.png">


Can you confirm this issue please?

**Reply by ibrunotome** [reply]
Nevermind, huge cache in safari 😆 Thanks David!

**Reply by juse-less** [reply]
@davidhemphill Thank you for looking into it. Can't wait for Tuesday/next release, if its added. ❤️

The problem, on top of lack of creativity and English capabilities, is that we have ourselves as a client in our CRM feature, to log our time internally (non-client stuff).
So.. now we have some interesting naming like.. internal client, client..client (?!), etc.
This is just a simplification.
We use lenses to create various reports for everything from time logs, invoicing, issues, etc.
So, when we now start creating them for 'internal clients', and 'client clients' (from example above), we get some interesting (and long) names.....

So, again, thank you so much for looking into making that change. That'd help me a lot sending out new features, while figuring out better naming. ❤️
